source/OfxSharp/OFXHelperMethods.cs

```diff
@@ -34,9 +34,15 @@ public static DateTime ToDate(this string date)
                 if (date.Length < 8)
                     return new DateTime();
 
-                var dd = int.Parse(date.Substring(6, 2));
-                var mm = int.Parse(date.Substring(4, 2));
-                var yyyy = int.Parse(date.Substring(0, 4));
+                int dd;
+                int.TryParse(date.Substring(6, 2), out dd);
+                int mm;
+                int.TryParse(date.Substring(4, 2), out mm);
+                int yyyy;
+                int.TryParse(date.Substring(0, 4), out yyyy);
+
+                if (yyyy == 0 || mm == 0 || dd == 0)
+                    return new DateTime(1, 1, 1);
```

Comment:
I'm wondering if it's better to return null instead 01/01/0001 because it's more easy to figure if we really have this data or not. I'm thinking in a scenario that the developer will record this data in database which this date is not suporting... instead changing to initial date is better to declare a nullable column. (It's just a suggestion, please feel free to implement it or not - just let me know for merge the PR or wait for you to implement).

Reply:
I agree with you and make same changes in code. Sorry for return only now, I need adjust my github account to send me e-mails from discussions.